mungegithub/mungers/submit-queue.go

```diff
@@ -647,6 +657,34 @@ func (sq *SubmitQueue) getQueueStatus() []byte {
 	return sq.marshal(status)
 }
 
+// getMetrics returns a json representation of important stats about the
+// submit queue's health, for external monitoring/graphing/alerting.
+func (sq *SubmitQueue) getMetrics() []byte {
+	sq.Lock()
+	defer sq.Unlock()
+
+	status := sq.e2e.GetBuildStatus()
+	blocked := false
+	for _, state := range status {
+		if state.Status != "Stable" {
+			blocked = true
+		}
+	}
```

Comment:
My question is, should this be more quickly/cleanly exposed in e2e/, rather than putting your type of brains into submit-queue.go